ambry-admin/src/main/java/com.github.ambry.admin/GetReplicasHandler.java

```diff
@@ -86,7 +86,7 @@ ReadableStreamChannel getReplicas(String blobId, RestResponseChannel restRespons
    */
   private JSONObject getReplicas(String blobId) throws RestServiceException {
     try {
-      PartitionId partitionId = new BlobId(blobId, clusterMap).getPartition();
+      PartitionId partitionId = BlobId.fromStringId(blobId, clusterMap).getPartition();
```

Comment:
There was some discussion that there should be a static method that does the des job, and building from scratch using constructor. I guess no such rule exists, so I will change it to constructor.